gc/gc.h

```diff
@@ -88,6 +91,7 @@ hash_replace_ref_value(st_data_t *key, st_data_t *value, st_data_t argp, int exi
     return ST_CONTINUE;
 }
 
+RBIMPL_ATTR_MAYBE_UNUSED()
```

Comment:
Shouldn't the `RBIMPL_WARNING_IGNORED(-Wunused-function)` above suppress warnings if this is unused?